src/main/java/org/fundacionjala/coding/manuel/Kata1.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.coding.manuel;
+
+/**
+ * Created by Administrator on 2/22/2018.
+ */
+public class Kata1 {
+    private String sentence;
+    private static final int MAX_SIZE = 5;
+
+    /**
+     *@param w Constructor by default.
+     */
+    public Kata1(String w) {
+        this.sentence = w;
+    }
+    /**
+     *@return Method to spin the sentence or word.
+     */
+    public String spinWord() {
+        String[] words = sentence.split(" ");
+        String newSentece = "";
+        StringBuilder newWord;
+        StringBuffer buffer = new StringBuffer();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Replace the synchronized class "StringBuffer" by an unsynchronized one such as "StringBuilder". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1149)